src/org/zaproxy/zap/extension/tokengen/TokenGenMessagesTableModel.java

```diff
@@ -117,6 +117,15 @@ public Object getValueAt(int rowIndex, int columnIndex) {
         }
     }
 
+    @Override
+    public boolean isCellEditable(int rowIndex, int columnIndex) {
+            if(getColumnClass(columnIndex) == String.class) {
+                return true;
+            }
+
+        return super.isCellEditable(rowIndex, columnIndex);
```

Comment:
True, however I prefer to use this approach in case a rewriting is needed and the AbstractTableModel is changed to something else.